vllm/core/block/prefix_caching_block.py

```diff
@@ -305,15 +326,14 @@ def get_common_computed_block_ids(
         Used in prefill (can skip prefill of some blocks).
         """
 
-        # TODO: Track computed blocks.
-        computed = lambda block_id: False
-
         # NOTE We exclude the last block to avoid the case where the entire
         # prompt is cached. This would cause erroneous behavior in model
         # runner.
+
         ids_list = [
-            takewhile(lambda block_id: computed(block_id), seq[:-1])
-            for seq in seq_block_ids
+            list(
+                takewhile(lambda block_id: self._blocks[block_id].computed,
+                          seq[:-1])) for seq in seq_block_ids
```

Comment:
This should have a unit test. cc @SageMoore .

I won't block the PR on it because you didn't add this code yourself, but it would be great to add.

Reply:
I see, would add a test case for it